### PR TITLE
no sudo means password failure on cert-based aws instance

### PR DESCRIPTION
**Pull request by mcfnord**
In this PR I add **sudo** to the second command of the compound bash line. Without this, I'm asked for a password on aws cert-based Ubuntu. With it, all is well. But I can't confirm this is best practice in all cases. Can you?

I also change _amend_ to _change_ as amending has a specific meaning that probably doesn't apply here.

### CODE REVIEW

File: wiki/en/Running-a-Server.md [hunk 1]
```diff
@@ -71,9 +71,9 @@ To run a headless server on Linux, the following steps assume you are familiar w
 
 1. Reload the systemd files and restart the headless Server:
 
-	`sudo systemctl daemon-reload && systemctl restart jamulus-headless`
+	`sudo systemctl daemon-reload && sudo systemctl restart jamulus-headless`
 
-_To amend your Server configuration, just repeat the last two steps above._
+_To change your Server configuration, just repeat the last two steps above._
```
Comment (pljones) [inline wiki/en/Running-a-Server.md]: As `systemctl edit` says "edit", we might as well use that, perhaps?
```suggestion
_To edit your Server configuration, just repeat the last two steps above._
```

**Review by pljones — APPROVED** [review]

**Comment by ann0see** [comment]
Looks good. I believe it can go to the release branch directly.

**Comment by jujudusud** [comment]
Is this needs to translated ?

**Comment by ann0see** [comment]
Yes, but it's a command so it will be translated to the exact same string.

**Comment by pljones** [comment]
> Yes, but it's a command so it will be translated to the exact same string.

Well, there's an explanation in plain English, too.

**Comment by ann0see** [comment]
Oops. We might want to do an intermediate update of the website anyway for fixed spelling mistakes etc.
